### ISSUE TITLE
Caller Announcement TTS

### ISSUE THREAD

**Issue by roundi** [jroal/a2dpvolume#258]
Hello,
Great and useful app, thanks
Is it possible to hear the caller name using TTS ?
Cheers,
Ronald

**Comment by jroal** [comment]
Already an Android feature.  Look at your Android accessibility features to find the caller ID readout.

**Comment by roundi** [comment]
Thank for your feedback. This feature was removed on later Samsung phone like s7 , it would be great to have. Txs

**Comment by jroal** [comment]
Yeah, I can't find it on my Motorola either. I agree this would be a good feature. Actually I think you can use the dialer notification with my apps notification reader. I will test this.

**Comment by roundi** [comment]
I found a function to speak the caller name, may be this will help : 
https://stackoverflow.com/questions/22552965/how-can-i-speak-a-callers-name-when-the-phone-is-ringing
I am not familiar with android coding, I wish I could help more.

**Comment by jroal** [comment]
I was able to set up the notification ready (just configuration of current app) to respond to the phone. I found 2 issues with this approach: today the message reader only sees "incoming call"; and it does not read messages while the phone is ringing.  However, after the call stops ringing A2DP Volume will read out who tried to call.  I know this is less useful as you want to know who was calling to decide if you want to answer.  I don't think I will be able to solve the problem of not being able to read out messages while the phone is ringing though.  More investigation is needed here....
